sklearn/linear_model/tests/test_passive_aggressive.py

```diff
@@ -125,6 +126,77 @@ def test_classifier_undefined_methods():
         assert_raises(AttributeError, lambda x: getattr(clf, x), meth)
 
 
+def test_class_weights():
+    # Test class weights.
```

Comment:
@vene, there's certainly things that may need to be factored over to common tests for the class/sample wt classifiers, seems like a different PR to me though... ping @amueller ?